cla_backend/apps/cla_butler/tests/test_tasks.py

```diff
@@ -7,10 +7,16 @@
 from core.tests.mommy_utils import make_recipe
 from cla_auditlog.models import AuditLog
 from complaints.models import Complaint
-from legalaid.models import Case, EODDetails
+from legalaid.models import Case, EODDetails, PersonalDetails
 
 
 class TasksTestCase(TestCase):
+    """
+    Currently in Django v1.7 these tests anything with an M2M relationship will not be tested correctly
+    Due to the DB of the tests not being a persisted.
```

Comment:
While I think I can see where this is coming from, I think it would be better to express it more directly: as the constraints not being checked inside the database transaction that gets rolled back at the end of each test